taxi/views.py

```diff
@@ -28,6 +30,16 @@ def index(request):
     return render(request, "taxi/index.html", context=context)
 
 
+def assign_driver(
```

Comment:
the name of the view is confusing, because its actually not only assigning drivers, but deleting them as well.

 you can simplify the logic and avoid passing the operation. you can check whether  request.user is already in  in car.drivers.all()  and add or delete them accordingly